components/gitpod-db/scripts/generate-latest-migration.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+# Licensed under the GNU Affero General Public License (AGPL).
+# See License.AGPL.txt in the project root for license information.
+
+get_latest_migration() {
+    # List all migrations and sort
+    migrations=$(find ./src/typeorm/migration/*.ts | sort -n | sed 's/\.\/src\/typeorm\/migration\///g' | sed 's/\.ts//g')
```

Comment:
to my understanding does it sort by TIMESTAMP?

Reply:
Does typeorm has some builtin utility to look up the latests migration?

Reply:
> Does typeorm has some builtin utility to look up the latests migration?

@akosyakov  Yes, it does: one is to use a connection that could get migrations, another is to align how they get migrations but without connection. You can find both code in old commits [here](https://github.com/gitpod-io/gitpod/pull/18455/commits/9589785d381e1b3cca77129a58ff539853b0c974)

It's good that we use connection one since we always have test-db init-ed. But it would be easier to build/run without any of them see https://github.com/gitpod-io/gitpod/pull/18455#discussion_r1287646319